.github/workflows/update.yml

```diff
@@ -8,17 +8,26 @@ jobs:
     name: Deploy docs
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout main
+      - name: Checkout
         uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Checkout LFS objects
+        run: git lfs pull
 
       - uses: actions/setup-python@v2
         with:
           python-version: 'pypy-2.7'
 
-      - name: Build pages and publish to S3
-        uses: dmitry-spodarets/mkdocs-s3-publish-action@main
+      - name: Build pages
+        uses: Exabyte-io/mkdocs-s3-publish-action@main
+
+      - name: S3 Sync
+        uses: unfor19/install-aws-cli-action@master
+      - run: aws s3 sync site/ s3://docs.mt3ra.com/
```

Comment:
Didn't the previous action already include the publish step? https://github.com/Exabyte-io/mkdocs-s3-publish-action/commit/6d96ff626877d167e5b578fa645ce95d61ef31e1

Reply:
From mkdocs-s3-publish-action the sync step has been removed due to minio client, so we have the next step for that. It works better this way.

Reply:
GA added to all pages automatically